test/runtests.jl

```diff
@@ -3,9 +3,25 @@ using Test
 
 import Dates
 import JSON3
+import Logging
 import URIs
 
+# This is necessary to work around https://github.com/JuliaLang/julia/issues/52234
+# Also possibly related: https://github.com/JuliaLang/julia/issues/34037
```

Comment:
Okay, for now, let's stick with LogLevels >= -1000.

https://github.com/JuliaHealth/FHIRClient.jl/pull/155

However, long-term, I think I'll want to revisit this. I think it would be unfortunate if the entire Julia ecosystem were restricted to only working with that narrower range of log levels, and I think it would be better if more Julia packages logged at a wider range of debug log levels, instead of just always defaulting to -1000.